types/zingchart/tslint.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "dtslint/dt.json"
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "npm-naming": [true,{"mode":"code","errors":[["NoDefaultExport",false]]}]
+  }
```

Comment:
You should be able to delete this now.

Reply:
Ah, thanks for getting back to me so quickly. I will apply this change.